roles/ironic-data/files/map-file

```diff
@@ -0,0 +1,2 @@
+r ^([^/]) /tftpboot/\1
+r ^(/tftpboot/) /tftpboot/\2
```

Comment:
Convention in Urusla so far has been to put EVERYTHING we copy onto a host machine into the templates/ directory tree, and use the `template` module for it. Any reason why this shouldn't follow that convention?